react-native-libraries.json

```
@@ -12635,5 +12628,46 @@
     "android": true,
     "web": true,
     "expoGo": false
+  },
+  {
+    "githubUrl": "https://github.com/margelo/react-native-nitro-sqlite",
+    "npmPkg": "react-native-nitro-sqlite",
+    "examples": [
+      "https://github.com/margelo/react-native-nitro-sqlite/tree/main/example"
+    ],
+    "images": ["https://github.com/margelo/react-native-nitro-sqlite/blob/main/header.png"],
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expoGo": false,
+    "windows": false,
+    "macos": true,
+    "tvos": false,
+    "visionos": false,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
+    "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/margelo/react-native-quick-sqlite",
+    "npmPkg": "react-native-quick-sqlite",
+    "examples": [
+      "https://github.com/margelo/react-native-nitro-sqlite/tree/react-native-quick-sqlite/example"
+    ],
+    "images": ["https://github.com/margelo/react-native-nitro-sqlite/blob/react-native-quick-sqlite/header2.png"],
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expoGo": false,
+    "windows": false,
+    "macos": true,
+    "tvos": false,
+    "visionos": false,
+    "unmaintained": true,
+    "dev": false,
+    "template": false,
+    "newArchitecture": true,
+    "alternatives": ["react-native-nitro-sqlite"]
```

Let's add only a new package at the end of file, and cleanup the entry a bit.

```suggestion
  {
    "githubUrl": "https://github.com/margelo/react-native-nitro-sqlite/tree/main/package",
    "npmPkg": "react-native-nitro-sqlite",
    "examples": [
      "https://github.com/margelo/react-native-nitro-sqlite/tree/main/example"
    ],
    "ios": true,
    "android": true,
    "macos": true,
    "newArchitecture": true
  }
```

CI was failing, since the `githubUrl` in monorepos needs to point out to the directory where `package.json` file for a given package is located.